File: backend/src/hono/routes/purge_test/purge_test.ts
import { Hono } from 'hono';
import { Context } from 'hono';

const app = new Hono();

app.post('/purge_test', async (c: Context) => {
  try {
    // Implementar lógica de teste de purga aqui
    return c.json({ message: 'Teste de purga executado com sucesso' }, 200);
  } catch (error) {
    return c.json({ message: 'Erro ao executar teste de purga' }, 500);
  }
});

export default app;

File: lib/features/purge_test/purge_test.dart
import 'package:flutter/material.dart';
import 'package:flutter_riverpod/flutter_riverpod.dart';

class PurgeTest extends ConsumerWidget {
  @override
  Widget build(BuildContext context, WidgetRef ref) {
    return Scaffold(
      appBar: AppBar(title: Text('Teste de Purga')),
      body: Center(
        child: ElevatedButton(
          onPressed: () async {
            // Implementar lógica de teste de purga aqui
          },
          child: Text('Executar Teste de Purga'),
        ),
      ),
    );
  }
}
